src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -29,17 +30,20 @@
 constexpr char qrCodeBaseUrl[]                   = "https://dhrishi.github.io/connectedhomeip/qrcode.html";
 constexpr char specialCharsUnreservedInRfc3986[] = "-._~";
 
-void PrintQRCode(chip::RendezvousInformationFlags rendezvousFlags)
+using namespace ::chip::DeviceLayer;
+
+void PrintOnboardingCodes(chip::RendezvousInformationFlags rendezvousFlags)
```

Comment:
Can we call GetSetupPayload once, and pass it into both GetQRCode and GetManualPairingCode?

Reply:
Sure, I can do that and I got your point, but I was wondering isn't it better to call it twice for the printing method than creating local variable to initialize setup payload outside of the method, calling GetSetupPayload every time before GetQRCode and so on? `PrintOnboardingCodes()` is kind of debug and in the release code it will not be used by anyone, while GetQRCode is used in many places like for generating NFC tag payload, before printing QR code on LCD and etc.